htdocs/vendor/js/mf-conditional-fields.js

```diff
@@ -0,0 +1,487 @@
+/**

```

Comment:
Hi @laemtl and @driusan, should this file be in htdocs/vendor/js or htdocs/js?

Reply:
I am confused by the difference between the two. Usually vendor/ stores libraries from external developers and js/ stores  code written internally.

Reply:
External things should be in vendor but it's possible that there are some leftover things from before we created the directory directly in js.